editor/src/clj/editor/defold_project_search.clj

```diff
@@ -51,7 +51,9 @@
   [{:keys [content resource] :as save-data}]
   (or (when (some? content)
         (make-line-coll #(BufferedReader. (StringReader. content))))
-      (when (and (resource/exists? resource) (not (text-util/binary? resource)))
+      (when (and (resource/exists? resource)
+                 (resource/textual? resource)
+                 (not (text-util/binary? resource)))
```

Comment:
If `(:textual? resource-type)` is `true`, we don't need to check if `(text-util/binary? resource)`, right? If I understand it correctly, we only need to do that if `resource-type` is `nil`?

Reply:
Hmm, if there is no resource type, `resource/textual?` will return true, but I see what you mean, we shouldn't check the contents of resources with specified types...